frontend/src/static/css/v_authorship.scss

```diff
@@ -91,8 +91,11 @@
       margin-top: 1rem;
       padding: .3em .5em;
 
-      .path {
+      .collapsable {
```

Comment:
collapsable -> collapsible

Reply:
is it possible to use 1 css class only instead of 2 (path and collapsible) ?

Reply:
> is it possible to use 1 css class only instead of 2 (path and collapsible) ?

Yep it is very possible. Thanks for the suggestion. I changed to that.